lib/pbench/test/unit/agent/test_tool_meister.py

```diff
@@ -57,37 +82,37 @@ class TestCreateTar:
         "tools": [],
     }
 
-    def test_create_tar(self, agent_setup, mock_tar):
+    def test_create_tar(self, agent_setup, mock_tar, caplog):
```

Comment:
Do you need either the `caplog` fixture or the `self.logger.setLevel` here? This is a success case, right; so there shouldn't be any warnings?